scripts/update-apibuilder.rb

```diff
@@ -29,13 +29,15 @@ def extract_version(path)
 path = File.join(dir, "../lib/flow_commerce/flow_api_v0_client.rb")
 checkout_version(`sem-info tag latest`)
 current = extract_version(path)
-checkout_version("master")
```

Comment:
Being honest, I am not sure the details behind this script. But shouldn't we keep `master` here? or is it referencing to the `main` branch from flow_commerce_spree gem??

Reply:
As we're not the ones responsible for the official `flowcommerce` gem, this is not a concern for us. The Flow team uses this to build a release I'd figure.